### DISCUSSION TITLE
Suitable DIN MIDI interface

### DISCUSSION THREAD

**Discussion by craigyjp** [probonopd/MiniDexed#154] [Show and tell]
I have built this MIDI interface for the UART connections, provides midi in/out/thru and indication of data in/out but you can tailor it to suit your needs.

![midi](https://user-images.githubusercontent.com/51026305/165546969-7aae9137-50c5-41ee-8809-0ae64c4171b2.jpg)

**Comment by luisfcorreia** [comment]
In principle, that should work.

(I would recommend using a 74ls14 (schmitt trigger inverter) to help shape up the signal path)

**Comment by craigyjp** [comment]
In principle it does work, I'm using it. I did consider the hc14 but went
with the ls245 as I'd already soldered the 20 pin socket.
